ja/lessons/basics/functions.md

```diff
@@ -1,5 +1,5 @@
 ---
-version: 1.2.0
+version: 1.2.1
```

Comment:
This version is for managing diff between origin lesson(en) and translated one.
Hence,  you don't need to bump up version if just typo fix which caused by translation)

Reply:
 I got it👍️